.github/workflows/wpcloud.yml

```diff
@@ -41,55 +41,43 @@ jobs:
         run: |
           CHANGED="$(EXTRA=test .github/files/list-changed-projects.sh)"
 
-          WPCOMSH_CHANGED="$(jq --argjson changed "$CHANGED" -n '$changed | has( "plugins/wpcomsh" ) ')"
+          # WPCOMSH_CHANGED="$(jq --argjson changed "$CHANGED" -n '$changed | has( "plugins/wpcomsh" ) ')"
+          WPCOMSH_CHANGED="true"
           echo "wpcomsh=${WPCOMSH_CHANGED}" >> "$GITHUB_OUTPUT"
 
       - name: Configure Github to be able to SSH to the Atomic site
         if: steps.changed.outputs.wpcomsh == 'true'
         run: |
-          echo "Intializing"
-          echo "::group::setup"
+          echo "::group::Intializing"
 
           mkdir -vp ~/.ssh/
           chmod -v 700 ~/.ssh
 
           touch ~/.ssh/id_site
+          touch ~/.ssh/known_hosts
           chmod 600 ~/.ssh/id_site
-          echo "$WPCLOUD_SSH_KEY" > ~/.ssh/id_site
+          chmod 600 ~/.ssh/known_hosts
+          echo "$SSH_KEY" > ~/.ssh/id_site
           echo "wrote ~/.ssh/id_site"
+          echo "$SSH_KNOWN_HOSTS" > ~/.ssh/known_hosts
+          echo "wrote ~/.ssh/known_hosts"
 
-          touch ~/.ssh/askpass
-          chmod -v 700 ~/.ssh/askpass
-          cat >>~/.ssh/askpass <<END
-          #!/bin/bash
-          echo "\$WPCLOUD_SSH_KEY_PASSPHRASE"
-          END
-          echo "wrote ~/.ssh/askpass"
-
-          cat >>~/.ssh/config <<END
-          Host *
-            StrictHostKeyChecking no
-          END
-          echo "wrote ~/.ssh/config"
+          echo "::endgroup::"
 
-          touch ~/.ssh/stdin
-          chmod -v 600 ~/.ssh/stdin
-          cat >>~/.ssh/stdin <<END
-          $WPCLOUD_STDIN
-          END
-          echo "wrote ~/.ssh/stdin"
+          echo "::group::Installing and building wpcomsh"
+          pnpm jetpack build plugins/wpcomsh
```

Comment:
You probably need to include `--deps`, and adding `-v` would probably be helpful in case anything fails.

Personally, I'd do the build as a separate step, if not as a separate job to make re-runs not have to re-build (cf. #32686).